releasenotes/notes/fix-qpy-symengine-compat-858970a9a1d6bc14.yaml

```
@@ -0,0 +1,62 @@
+---
+fixes:
+  - |
+    Fixed an issue with :func:`.qpy.load` when loading a QPY file containing
+    a :class:`.ParameterExpression`, if the versions of ``symengine`` installed
+    in the generating and loading environments were not the same.  For example,
+    if a QPY file containing :class:`.ParameterExpression`\ s was generated
+    using Qiskit 1.2.2 with ``symengine==0.11.0`` installed, Qiskit 1.2.2 with
```

```suggestion
    using Qiskit v1.2.2 with ``symengine==0.11.0`` installed, Qiskit v1.2.2 with
```
using v before version number was prescribed by the editorial team - not sure how urgent it is to apply it in API docs

---

If this is a chance the editorial team want, we might want to do it all in one go in a follow up - if I do this here, the rest of the Qiskit 1.2 release notes page (so 1.2.2, 1.2.1 and 1.2.0) will still be using the old form, so it might look disjointed.